src/components/FlawTrackers.vue

```diff
@@ -302,9 +303,16 @@ function increaseItemsPerPage() {
               :key="trackerIndex"
             >
               <td>
-                <RouterLink :to="{ path: `/tracker/${tracker.uuid}` }">
+                <a
+                  v-if="tracker.external_system_id"
+                  :href="trackerUrl(tracker.type, tracker.external_system_id)"
+                  target="_blank"
+                >
                   {{ `${tracker.external_system_id} ` }}<i class="bi-box-arrow-up-right" />
-                </RouterLink>
+                </a>
+                <span v-else title="This tracker doesn't have External ID">
+                  Undefined
```

Comment:
I would prefer to use `--` or just empty when there's no ID, `Undefined` seems like a bug as we don't use it anywhere else

Reply:
Empty was my first option, but when checking the UI it was feeling weird.

I understand that `undefined` feels like a bug to developers, but I think having empty or random characters can be more confusing to the users than 'undefined', cause it provides direct self-explanatory information.

Not strong preference.

Reply:
Its just weird, we don't use it anywhere and to me it seems like the link is broken or something happened and the value is "Undefined" instead of not having a link, maybe the word "None" is better if you don't like the `--`, other than that I think it looks good so I'll give my approve

Reply:
Let'g go for `None`